umibukela/views.py

```diff
@@ -46,11 +45,40 @@ def site(request, site_slug):
     })
 
 
+def site_result_hardcoded(request, site_slug, result_id):
+    result_set = CycleResultSet.objects.get(
+        id=result_id,
+        site__slug__exact=site_slug
+    )
+    return render(request, 'site_result_hardcoded.html', {
+        'active_tab': 'sites',
+        'result_set': result_set,
+    })
+
+
 def site_result(request, site_slug, result_id):
-    result_set = CycleResultSet.objects.get(id=result_id, site__slug__exact=site_slug)
+    result_set = CycleResultSet.objects.get(
```

Comment:
What if the slug or id are invalid? Django has a get_object_or_404 helper which is useful.